aea/crypto/ethereum.py

```diff
@@ -221,6 +221,12 @@ def send_transaction(
             "gasPrice": self._api.toWei(GAS_PRICE, GAS_ID),
             "data": tx_nonce,
         }
+        gas_estimation = self._api.eth.estimateGas(transaction=transaction)
+        assert (
+            tx_fee >= gas_estimation
+        ), "Need to add more gas. Change the configs. Estimated Gas is: {}".format(
```

Comment:
Change to "Need to increase `tx_fee` in the configs to cover the gas consumption of the transaction. Estimated gas consumption is: {}."